### ISSUE TITLE
Allow multiple entries in graffiti file

### ISSUE THREAD

**Issue by rolfyone** [Consensys-Incorporated/teku#3471]
Now that graffiti can be loaded from file, we should consider allowing multiple entries in the graffiti file.

Prysm use cases: https://docs.prylabs.network/docs/prysm-usage/graffiti-file/

There's some interesting use-cases listed where randomising, or having specific per-validator graffiti might help obfuscate validators being run, or indeed for the graffiti wall.

It's worth having a discussion about use-cases and a proposed solution.

https://discord.com/channels/697535391594446898/697539289042649190/801199770956726282

**Comment by devax** [comment]
+1. Would love to have the "ordered" option. This would enable the possibility to actually paint something on the graffiti wall.

**Comment by vincenttaglia** [comment]
+1. I think this would be a fun feature that also has the potential to encourage client diversity. Many solo stakers that are paying attention to things like client diversity like to personalize their validator with graffiti, so adding functionality like Prysm has would be one less reason to stick with Prysm.

**Comment by tbenr** [comment]
This can be integrated in proposer configuration file. See `--validators-proposer-config` parameter.

**Comment by ameya-deshmukh** [comment]
@rolfyone is this issue still open? If so, I'd love to pick it up

**Comment by rolfyone** [comment]
> @rolfyone is this issue still open? If so, I'd love to pick it up

Yes still open! I haven't really given it any thought yet to be honest, but shout if you need some guidance and we can assist! There's a #teku-contributors chan on  Consensys discord also if you have questions we can help with.... 

Issue also works obviously :) just making sure you've got some starting points of contact :)

**Comment by ameya-deshmukh** [comment]
Awesome! Joining in a bit :D

**Comment by LefterisJP** [comment]
Just adding my proverbial +1 request for this feature :pray:
